docs/blog/2019-12-09-inferring-from-history/index.md

```diff
@@ -0,0 +1,76 @@
+
+So what could we infer from “history” and the “environment” to solve the problem we had in Gatsby Cloud, where everyone was confused and frustrated at needing to choose a GitHub organization?
+
+### Inferring from history
+The first step is simply to look at history, at what happened before the point of confusion. I’ll take you on a tour of what happened just moments before users hit their point of major confusion.
+
+In this first screenshot, the user selects a starter.
+![Screenshot of the starter page](final-state.png)
+
+Then, they log into GitHub and give Gatsby Cloud permission to connect with their personal GitHub account. 
+![Screenshot of the Gatsby Cloud login page with a thought bubble the author drew on top of the screenshot. The thought bubble contains the text “Alright, I can sign in with Git Hub. I have an account!”](cloud-login.png)
+
+Then, this screen asks them to “add new organization.” This is where the software failed to learn from recent history. The user just gave the system access to their personal GitHub account, so that the last value they gave the system and we ought to stick with that value.
+
+![Screenshot of the landing page for first-time visitors who have logged into Gatsby Cloud with a thought bubble the author drew on top of the screenshot. The thought bubble contains the text “What organization? GitHub organization? If so, why would I need to connect to GitHub again?” and "Gatsby Cloud, you have left me with no choice...I must click this button" referring to the "Add New Organization" button](add-new-org-confusion.png)
+
+By the time the user adds an organization (and they don’t know why they have to add it), and sees this next screen below, of course they are confused about why they added an organization. They didn’t need to!
+ 
+![screenshot of step two of Gatsby Cloud onboarding, during which users select an organization from GitHub. I drew a speech bubble on top of the screenshot pointing out that users are confused by this screen. The speech bubble says: “Wait, I don't want to create a repository in my work's GitHub organization. Terminate mission!”](select-work-org-confusion.png)
+
+To reduce interaction by inferring from history, we assume the user wants to save their first site in their GitHub personal account, the last value they provided us with when they logged in. They can *correct* this assumption if it’s wrong.
+
+This new assumption is reflected in the following GIF, which depicts a screen in which the user names their site and clicks “Next." The user does *not have to interact with the software to tell us where to save their site*, though they *can correct the default if it’s wrong*, because the text on the screen reads: “we’ll create the repo under your personal GitHub account. Want to host it on a GitHub organization instead? Add it here!”
+
+![GIF](create-new-site.gif)
```

Comment:
Same comment as the "Screenshot" alt text above.

Reply:
Fixed! Thank you!